lib/flutter_svg_provider.dart

```diff
@@ -111,20 +111,24 @@ class Svg extends ImageProvider<SvgImageKey> {
   }
 
   static Future<ImageInfo> _loadAsync(SvgImageKey key) async {
-    final String rawSvg = await _getSvgString(key);
+    final rawSvg = await _getSvgString(key);
     final pictureInfo = await vg.loadPicture(
       SvgStringLoader(rawSvg),
       null,
       clipViewbox: false,
     );
-    final ui.Image image = await pictureInfo.picture.toImage(
-      pictureInfo.size.width.round(),
-      pictureInfo.size.height.round(),
-    );
+    final image = kIsWeb
+        ? await pictureInfo.picture.toImage(
+            pictureInfo.size.width.round(),
+            pictureInfo.size.height.round(),
+          )
+        : pictureInfo.picture.toImageSync(
+            pictureInfo.size.width.round(),
+            pictureInfo.size.height.round(),
+          );
 
     return ImageInfo(
       image: image,
-      scale: 1.0,
```

Comment:
Why was this removed?

Reply:
The value of the argument is redundant because it matches the default value. [(Documentation)](https://dart.dev/tools/linter-rules/avoid_redundant_argument_values)